appcircle_publish_send_to_testflight/1.2.1/component.yaml

```diff
@@ -0,0 +1,39 @@
+platform: iOS
+purpose: Publish
+marketPlace: "AppleAppStore"
+buildPlatform:
+displayName: "Send to TestFlight"
+description: "Upload binary for TestFlight publishing"
+docsUrl: https://docs.appcircle.io/publish-integrations/ios-publish-integrations/sent-to-testflight
+requiredComponents: 
+precedingComponents: "appcircle_publish_appstore_app_information"
```

Comment:
Why we need to add this step after the appcircle_publish_appstore_app_information step?